bokeh/server/session.py

```diff
@@ -176,29 +177,12 @@ def _remove_timeout_callback(self, callback):
         self._loop.remove_timeout(cb)
 
     def _document_changed(self, event):
+        pass
```

Comment:
hehe... now I see we were saying the same things on the previous conversation. Will do it